README.md

```diff
@@ -233,6 +235,21 @@ Need to setup Flatpak (https://flatpak.org/setup/):
 * Chrome OS
 
 
+Systemd
+-------
+If your distribuition is running under systemd and you want to generate and upload
+hw-probe report autmaticaly, please install as root:
+
+    cp -a systemd/* $(systemdsystemunitdir)/
+
+Enable hw-probe.timer by running as root:
```

Comment:
Rename to hw-probe.monthly?

Reply:
Extenstion `.timer` is mandatory name. Ofcourse user may edit file and change `OnCalendar` value to daily, weekly etc..